src/main/java/ToDos.java

```diff
@@ -0,0 +1,24 @@
+public class ToDos extends Task {
+    protected String taskType = "[T]";
+
+    public ToDos(String description) {
+        super(description);
+    }
+
+    @Override
+    public void fullDesc() {
```

Comment:
It seems like the child classes are not able to take advantage of this method by calling ```super.fullDesc()```, perhaps you could tweak the method in ```Task``` slightly such that you can further abstract out the code?